terraform/lambda/issued-credentials.tf

```diff
@@ -0,0 +1,20 @@
+module "issued-credentials" {
+  source      = "../modules/endpoint"
+  environment = var.environment
+
+  rest_api_id            = aws_api_gateway_rest_api.ipv_internal.id
+  rest_api_execution_arn = aws_api_gateway_rest_api.ipv_internal.execution_arn
+  root_resource_id       = aws_api_gateway_rest_api.ipv_internal.root_resource_id
+  http_method            = "GET"
+  path_part              = "request-credentials"
```

Comment:
I'd suggest this be `issued-credentials` to keep the naming consistent.

Reply:
renamed path to issued-credentials